lms/static/scripts/frontend_apps/components/test/AppRoot-test.js

```diff
@@ -70,6 +70,7 @@ describe('AppRoot', () => {
     {
       config: { mode: 'email-notifications' },
```

Comment:
If the route is `/email/preferences` it might be a good idea to align the component and mode names with that in future.

Reply:
Yep, I also thought about that, I'll do in a follow-up PR.